odoo/addons/base/models/ir_fields.py

```diff
@@ -201,7 +202,10 @@ def to_field(self, model, field, fromtype=str):
         converter = getattr(self, '_%s_to_%s' % (typename, field.type), None)
         if not converter:
             return None
-        return functools.partial(converter, model, field)
+        kw = {}
+        if 'savepoint' in signature(converter).parameters:
+            kw['savepoint'] = savepoint
```

Comment:
Change the signature everywhere, it is easier for reasoning.

Reply:
Updated.